cfg/rationalize_test.go

```diff
@@ -664,3 +665,63 @@ func TestRationalize_FileCacheAndBufferedReadConflict(t *testing.T) {
 		})
 	}
 }
+
+func TestResolveLoggingConfig(t *testing.T) {
```

Comment:
Seems excessive to me to test for the logged string. Checking the format should be good enough. Looking for the log string makes it a change detector test.